lib/cli.mjs

```diff
@@ -319,6 +323,11 @@ const flags = {
 		shortFlag: 'v',
 		type: 'boolean',
 	},
+	computeEditInfo: {
```

Comment:
Let's put this after the `color` property to maintain the alphabetical order.

Reply:
Thanks for all the details. 🙇